_posts/2024-04-15-gsi-tech-blog.md

```diff
@@ -0,0 +1,75 @@
+
+Communications satellites are great at talking to each other, but earth observation satellites don’t always have such a luxury. They have large arrays of sensors, and collect large quantities of data. However they still have to send their data to the ground for processing. That is to say, these observational satellites don’t really compute on their own sensor data. To further complicate this issue, sending sensor data to earth has to happen in small windows of time, due to satellites only being able to transmit to ground stations in small windows in their orbit. This means that not all of their data is sure to make it. Even after all of this happens, a human still has to review the results.
+
+The number of satellites is increasing. We have five times more satellites in space than we did ten years ago. The proportional amount of humans we have to do this processing is decreasing. Humans have become a bottleneck.
+
+![](/assets/media/blog-images/2024-04-15-gsi-tech-blog/objects-launched-annual.jpg)
+
+## What if...
+
+Suppose these types of satellites could be outfitted with a card that both stored and computed, equivalent to having both RAM and CPU, with the capability to run their sensor data through an ML model. The goal being to vectorize and pre-process sensor data. Sending a vector or change detection earthside is a much quicker operation than sending a complete set of sensor data. One of our partners, [GSI Technology](https://gsitechnology.com), might just change some of the landscape in this regard. They’ve introduced a new kind of chip into the market. The project, called “Gemini-II”, is an APU. It is an extremely power efficient processing unit that can both store and compute in its own distributed memory. You may have seen their booth at [OpenSearchCon 2023](http://www.opensearch.org/events/opensearchcon/2023/north-america/index.html).
+
+The APU technology family is a Compute-in-memory (CiM) architecture seeking to eliminate computation time wasted on data movement by closing the traditional gap between processors (millions in GSI's case) and storage elements, which is highly beneficial for data-intensive workloads. Computation occurs where the data lives - in the storage elements, avoiding traditional Processor/Memory bottlenecks!
+
+## Several applications
+
+Many institutions often use things like SAR (synthetic aperture radar) to monitor landscapes to check for things like the progress of deforestation, forest fires, as well as the erosion of coastal areas. Even corporations use this tech to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use this tech to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.
```

Comment:
```suggestion
Many organizations use technology like synthetic aperture radar (SAR) to monitor landscapes to check for things like deforestation progress, forest fires, or erosion of coastal areas. Even corporations use this technology to count the number of cars in their stores’ parking lots to help forecast business decisions. We can even use it to predict and detect ships adrift at sea for the purposes of rescue. The same advantage is available here. Instead of sending entire collections of satellite data, we now have the potential to send results that represent those changes instead.
```